### DISCUSSION TITLE
I'm new to MiniShop2

### DISCUSSION THREAD

**Discussion by Jsewill** [Ibochkarev/miniShop2#517] [Q&A]
I'm new to miniShop2, and am enjoying using it to the degree that I can understand it from the available documentation. I have a few questions. I apologize if this is the wrong place to submit them. Here are my questions:

Is there any more documentation, or are there any tutorials, other than what can be found at [https://docs.modx.pro/en/components/minishop2](docs.modx.pro)?

Are there any addons available to the public, relating to:
- payment
- delivery
- configurable products (products that change price based on selected options)

Is there any english documentation relating to the creation of those things?

I assume the answer is yes to this one: If I make changes to the default chunks, will those changes be overwritten by a package upgrade?

That's all the questions I have for now. Thanks in advance for any answers or direction.

**Comment by Ibochkarev** [comment]
@Jsewill Good day!

Thank you for your interest in our component.

We will update the English translations of the current documentation base, after the update we will need help correcting errors. Since we are not native speakers.

**Reply by Jsewill** [reply]
Thanks for responding to my question. While I have you here, would you kindly answer another question I have? I haven't been able to figure it out: What exactly is the, "Article" product field? Is it meant to be a product identifier, such as a product number, model number, or SKU?

Also, is there a way to manage stock (even a simple in-stock/out-of-stock boolean), or to show a product without allowing "Add to cart" or purchasing it (such as a product that's "coming soon")? I'm certain I could write in some functionality using product options and the various templates and such, but I want to avoid as much undue modification as possible. Thanks for your patience!
